test/05_install_from_local_repo.py

```diff
@@ -8,13 +8,13 @@
 
 c.expect(r'([0-9]+)\. htop', timeout=10)
 entry_id = c.match.groups()[0]
-print('PEXPECT: Found entry id', repr(entry_id))
+print(f'PEXPECT: Found entry id', entry_id)
```

Comment:
I would keep the repr functionality for always proper quoting, but let's use f-strings with something like
```python
print(f'PEXPECT: Found entry id {entry_id!r}')
```